suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -811,10 +805,28 @@ export const selectDeviceNameById = (
     return device?.name ?? null;
 };
 
-export const selectDeviceLabel = (state: DeviceRootState) => {
-    const selectedDevice = selectDevice(state);
+export const selectDeviceLabel = (state: DeviceRootState): string => {
+    const device = selectDevice(state);
+
+    return device?.features?.label ?? '';
```

Comment:
Please don't use fallback to empty string or don't use `if("empty string")` later, it's confusing and too magical that empty string is false in JS

Reply:
ecc4143c4b862e3cc2ee9ca90503f5b881d476f9

Reply:
Exact `=== ''` comparison is less confusing, but still, `null` would be more semantic than `''`.

Could `selectDeviceLabel` return `string | null`? I see it is only used in two places: `ChangeDeviceLabel, DeviceInfoModalScreen`, everywhere else it's `selectDeviceLabelOrName` which would still return only `string`.
But maybe it's not worth it, we can discuss IRL :slightly_smiling_face:

Reply:
324f51237dc1ecd5bb34b65caf3e8e5b1a712206